src/lib/features/playground/advanced-playground.test.ts

```diff
@@ -11,7 +11,7 @@ let db: ITestDb;
 
 beforeAll(async () => {
     db = await dbInit('advanced_playground', getLogger, {
-        experimental: { flags: { strategyVariant: true } },
+        experimental: { flags: {} },
```

Comment:
do we need this to be explicitly here but empty?

Reply:
nope, removing